fmriprep/cli/run.py

```diff
@@ -330,6 +330,11 @@ def build_workflow(opts, retval):
             raise RuntimeError(msg)
         logger.warning(msg)
 
+    # Cannot output CIFTI if no freesurfer surfaces
+    if opts.cifti_output and not opts.run_reconall:
```

Comment:
instead of logging, could this return an error in the command line itself. `argparse` has xor type options.